handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+# Network Security
+
+SystemLink connects test systems to a central server to aggregate data for monitoring and analysis. SystemLink uses industry standard tools and best practices to ensure the SystemLink server and test systems utilize secure networking techniques.
+
+Refer to [Workspaces and Role-based Access Control](/rbac/rbac) for details secure user interactions with SystemLink resources. Refer to [Single Sign-on with OpenID Connect](/openid-connect/openid-connect/) and [Sign on with LDAP](/ldap/ldap/) for details on securely authenticating with SystemLink.
+
+## Network diagram
+
+<figure>
+  <img src="../../img/network-diagram.png" width="500" />
+  <figcaption>The networked components and encrypted protocols used by SystemLink.</figcaption>
+</figure>
+
+!!! note
+    Refer to [Setting up a SystemLink Server](https://www.ni.com/documentation/en/systemlink/2020r4/setup/setting-up-systemlink-server/) for information regarding ports used by SystemLink. d
+
+- **SystemLink App Server**:
+    - This Windows server hosts NI Web Server, the various SystemLink services, SystemLink web applications, and a Salt master.
+    - In the *single-box* setup, MongoDB and FIS storage are hosted on the app server.
+    - Refer to [Sizing a SystemLink Server](https://www.ni.com/documentation/en/systemlink/2020r4/setup/sizing-a-systemlink-server/) for details on server system requirements.
+    - NI assumes users with desktop access to the server are outside of the security controls and capabilities described by this document. Securing desktop access to the SystemLink app server should be managed by the customer's IT security protocols.
+
+- **SystemLink Web app**
+    - The SystemLink web application is the primary way users interact with SystemLink.
+    - NI recommends HTTPS for all production environments.
+    - Refer to [**NI Web Server**](#ni-web-server) for details on configuring encrypted communication.
+
+- **Targets**
+    - Targets are Windows and Linux RT systems who software and configuration are managed by SystemLink.
+    - Software configuration occurs via the [SaltStack Transport Protocol](https://docs.saltproject.io/en/latest/topics/transports/).
+        - Refer to [**Target to SystemLink Communication**](#target-to-systemlink-communication) for details regarding how SystemLink encrypts this communication.
+    - File, tag, test, and asset data are published to the SystemLink app server over HTTPS.
+
+- **MongoDB**
```

Comment:
Is it worth mentioning Redis or Postgres here? Not sure when the Postgres stuff is slated for submission.

Reply:
We haven't seen the demand for running Redis on a separate instance the way we have for MongoDB. Its likely we'll retire the tag implementation that uses Redis before we really need to document that configuration here. 

I expect we'll update this doc when PostGres is in the product.